src/provider/states/download_package_backoff.rs

```diff
@@ -0,0 +1,32 @@
+use kubelet::backoff::BackoffStrategy;
+use kubelet::state::prelude::*;
+use log::info;
+
+use crate::provider::PodState;
+use crate::provider::repository::package::Package;
+use crate::provider::states::download_package::Downloading;
+
+#[derive(Debug, TransitionTo)]
+#[transition_to(Downloading)]
+/// The Pod failed to run.
+// If we manually implement, we can allow for arguments.
```

Comment:
I don't understand this comment

Reply:
Leftover from the state I copied this from.